src/test/types/react-redux.js

```diff
@@ -62,7 +61,9 @@ class ExampleComponent extends React.PureComponent<Props> {
 
     // The action creators are properly wrapped by dispatch.
     (this.props.dispatchString: string => Action);
+    // $FlowFixMe Error introduced by upgrading to v0.96.0.
     (this.props.dispatchThunk: string => number);
+    // $FlowFixMe Error introduced by upgrading to v0.96.0.
```

Comment:
I find this a bit concerning because this is one of the root of our app architecture :/

Reply:
I bet we can fix it, but not in this upgrade. I got stuck here before, so I think it's better to regress this. See: #1936